content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -0,0 +1,157 @@
+kind
+```
+* kubectl
+ Install kubectl:
+ 
+ Download the latest release with the command:
+```bash
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+ Note:
+ 
+If you do not have root access on the target system, you can still install kubectl to the ~/.local/bin directory:
+```bash
+chmod +x kubectl
+mkdir -p ~/.local/bin/kubectl
+mv ./kubectl ~/.local/bin/kubectl
+# and then add ~/.local/bin/kubectl to $PATH
+```
+Test to ensure the version you installed is up-to-date:
+```bash
+kubectl version --client
+```
+### Build the delfin docker image locally
+Download the source code of delfin:
+```bash
+git clone https://github.com/sodafoundation/delfin.git
+```
+
+Build Delfin Image with k8s tag locally: 
+```bash
+cd delfin
+docker build -t sodafoundation/delfin:k8s .
+```
+
+### Get all kubernetes object files required for delfin deployment:
+```bash
+git clone https://github.com/sodafoundation/examples.git
+```
+### All the commands required to bring up the delfin services:
+Run this from the delfin directory.
+
+Create all the config Maps from files.
+```bash
+kubectl create configmap delfin-config --from-file=../delfin/etc/delfin/
+```
+Run this from the examples/delfin-kubernetes directory.
+Create all pods
+```bash
+kubectl apply -f delfin-api-deployment.yaml
+kubectl apply -f delfin-task-deployment.yaml
+kubectl apply -f delfin-exporter-deployment.yaml
+kubectl apply -f delfin-alert-deployment.yaml
+kubectl apply -f redis-deployment.yaml
+kubectl apply -f rabbitmq-deployment.yaml
+```
+Expose the services
+```bash
+kubectl apply -f delfin-api-service.yaml
+kubectl apply -f delfin-exporter-service.yaml
+kubectl apply -f delfin-alert-service.yaml
+kubectl apply -f redis-service.yaml
+kubectl apply -f rabbitmq-service.yaml
+```
+Get all the deployed objects
+```bash
+Kubectl get all
+```
+### To monitor the performance metrics on prometheus
+Follow the below site:
+
+https://devopscube.com/setup-prometheus-monitoring-on-kubernetes/
+
+### To send an API request to a service
+Use node IP and node port of the service
+
+Example:
+```bash
+#Run the GET API to get the registered storages.
+curl -X GET http://<IP of k8s node>:<nodePort of delfin-api-service SVC>/v1/storages
+#Will give all the registered storages
+```
+Note:
+For accessing cluster from outside, endpoint should be set which can be connected from outside the cluster
+### Deleting the delfin on k8s
+Delete all the config Maps from files.
+```bash
+kubectl delete configmap delfin-config --from-file=../delfin/etc/delfin/
+```
+Delete all pods 
+```bash
+kubectl delete -f delfin-api-deployment.yaml
+kubectl delete -f delfin-task-deployment.yaml
+kubectl delete -f delfin-exporter-deployment.yaml
+kubectl delete -f delfin-alert-deployment.yaml
+kubectl delete -f redis-deployment.yaml
+kubectl delete -f rabbitmq-deployment.yaml
+```
+Expose the services
```

Comment:
When deleting do we need to expose the services or is this a typo?

Reply:
its typo changed